website/oauth/views.py

```diff
@@ -41,17 +50,27 @@ def oauth_connect(service_name, auth):
 
 @must_be_logged_in
 def oauth_callback(service_name, auth):
-    user = auth.user
-    provider = get_service(service_name)
-
-    # Retrieve permanent credentials from provider
-    if not provider.auth_callback(user=user):
-        return {}
-
-    if provider.account and not user.external_accounts.filter(id=provider.account.id).exists():
-        user.external_accounts.add(provider.account)
-        user.save()
-
-    oauth_complete.send(provider, account=provider.account, user=user)
+    if waffle.flag_is_active(request, features.ENABLE_GV):
+        code = request.args.get('code')
+        state = request.args.get('state')
+        query_params = {
+            'code': code,
+            'state': state,
+        }
+        gv_url = urlunparse(urlparse(GRAVYVALET_URL)._replace(path='callback', query=urlencode(query_params)))
+        requests.get(gv_url)
```

Comment:
nit: extract this into a local utility function? `_forward_to_addon_service(request)` or the like?